plugins/copr.py

```diff
@@ -96,23 +97,21 @@ def configure(self, args):
 
 
         # Useful for forcing a distribution
-        raw_copr_plugin_config = ConfigParser()
+        copr_plugin_config = ConfigParser()
         config_file = None
         for path in self.base.conf.pluginconfpath:
             test_config_file = '{}/{}.conf'.format(path, PLUGIN_CONF)
             if os.path.isfile(test_config_file):
                 config_file = test_config_file
 
         if config_file is not None:
-            cp = raw_copr_plugin_config.read_file(config_file)
-            distribution = (cp.has_section('main')
-                            and cp.has_option('main', 'distribution')
-                            and cp.get('main', 'distribution'))
-            releasever = (cp.has_section('main')
-                          and cp.has_option('main', 'releasever')
-                          and cp.get('main', 'releasever'))
-            self.chroot_config = [distribution, releasever]
-
+            copr_plugin_config.read(config_file)
+            if copr_plugin_config.has_option('main', 'distribution') and copr_plugin_config.has_option('main', 'releasever'):
+                distribution = copr_plugin_config.get('main', 'distribution')
+                releasever = copr_plugin_config.get('main', 'releasever')
```

Comment:
This read distribution and releasever only when both are set. While previously it was:
```
  if copr_plugin_config.has_option('main', 'distribution')
      distribution = ...
  if copr_plugin_config.has_option('main', 'releasever')
      releasever = ...
```
Hmm but it does not matter, because i _guess_chroot we set it from platform.linux_distribution()
if one of them is not set, overriding the other. So I'm waiving out this difference.